docs/manual_works.rst

```diff
@@ -64,7 +64,7 @@ Manuscript Share
 ----------------
 
 Django-Music-Publisher uses a very simple single-field share model. The principle is very simple.
-Writers create a work and decide how they want to split the shares among themselves. This is referred to as
+Writers create work and decide how they want to split the shares among themselves. This is referred to as
```

Comment:
Why?
"Work" here is a shorthand for "muscial works", separate and countable objects.

Reply:
Actually you're right. It's with 'a'. It's definitely correct.